### ISSUE TITLE
What sort of guest speakers would you like to see? 

### ISSUE THREAD

**Issue by hectorsector** [campus-experts/fall-2016#1]
We want to invite some guest speakers to our upcoming office hours. What topics or sort of guest speakers would you like to see?

Below are some topics we're thinking about presenting. We want to know if these are topics you're interested in, and we want to hear you own suggestions or thoughts in this issue.

- Make a great social impact
- Brand your community
- Become a great public speaker
- Develop workshop content

**Comment by gillchristian** [comment]
For me all the topics seems interesting 😊

Personally I admire Kent C. Dodds. He is part of the JS community, a referent I'd say. He is a great speaker and does some killer workshops.

I'd like to learn more about that. How to find topics of interest and be able to use them to contribute to the community, either by making interactive workshops or inspiring talks.

**Comment by hackerkid** [comment]
It would be awesome to have a talk on building and branding the community.

**Comment by mchavana** [comment]
I'd really like to hear from somebody who has worked on developing a project or someone who is working on community contributions.

**Comment by gcttong** [comment]
3 speaker topics that would be helpful for our local community are:
1. Finding a speaker who has experience at persuading others in fighting for a common cause. How do we really sell ourselves and convince others that what we are trying to build is worthwhile?

2. How do we build sustainable hacker communities, one that will endure well after we leave our schools?

3. Ideas for workshops that provide value ? Ones that can be conducted within a short time slot, <2 hours and provide a good learning experience for attendees.

**Comment by ruxiang05** [comment]
It would be awesome if we could have:

-  someone who has had a hard time getting into developing, but overcame their situation
- session on how to get into open source development
- sessions on project management

**Comment by Lionex** [comment]
I'd love to hear about fundraising and sponsorship for organizations (not just large events like hackathons).

**Comment by sydko** [comment]
I second gcttong's comments and would also like to add:
- various speakers who have experience teaching to different audiences (eg. middle school students, to high school, to college, to new grads) and how they changed workshops to address the needs of different age groups.

**Comment by sara-h-chen** [comment]
I know this is a difficult one, but it'd be incredibly useful if we could attend a talk that would give us a rough idea of the legal obligations that we can expect to come up against when running tech events, especially large ones, and the ways that we can handle these obligations. :)

**Comment by AthenaRaya** [comment]
Some topics I think may be useful:
1.  how to contribute to open source development and knowing where to start
2. how to fundraiser with a balance of zero
3. how to Keep the passion and movement  of community to grow

**Comment by kim-codes** [comment]
It would be neat to have someone from the Community & Safety team give a talk on the steps we can take to foster a safe and open community (online or not).

**Comment by DickensA** [comment]
Here are some talks I think would be great to see for community building: 
1. How to deal with anxiety & support, Jess Rose (crate.io) does a great talk on imposter syndrome that discusses these things . 
2. Teams & Projects - my friend (a medic) has a great story about a responsibility pie which is so true when managing community events as part of a larger team - it would be nice to see something like this appear in a talk.
3. Any form of skills workshops are great to see happening and very welcoming to beginners.

**Comment by hectorsector** [comment]
Thanks for the names @DickensA! @joenash and I will follow up on these.

**Comment by hectorsector** [comment]
> Ideas for workshops that provide value ? Ones that can be conducted within a short time slot, <2 hours and provide a good learning experience for attendees.

@gcttong and @campus-experts/fall-2016: as part of the GitHub training team, I find that what I have to offer is very much in need when I visit groups. Would a session on how we teach Git and GitHub to others be a helpful one? We have a few resources and teaching philosophy we could leave you with if you'd like to hold this kind of workshop for your own groups -- but only if it's a need that enough of you have.

**Comment by gcttong** [comment]
@hectorsector Yes, I think having session on how to teach GitHub would be very helpful for our community, something I saw @sara-h-chen discuss about in the slack, definitely want to hear more about it, the idea for different skill levels also sounds really cool. I could see our group needing to have a Github/Git training session more than once during a semestre, and because it's such a valuable skill in the developer community.

**Comment by xoneco** [comment]
I'm quite late on this, but here's something I've been thinking about recently:

*Community Culture and the best way to build and maintain it.*

A lot of the code of conducts come down to: "Don't be a jerk", which I think is great but also leaves a lot of room to interpretation which is then effectively shaped by the people in the community. 
Even if a code of conduct attempts to define this in great detail, the longer a coc becomes, the fewer people will read and subsequently stick to it.

So a few questions emerge from that:
* How do you deal with people that technically don't break any coc rules but still go against the culture you are trying to promote. Do you speak to them? Privately or publicly? Do you rely on the community to self-regulate?
* Cultures need to develop organically, so how much steering do you do?
* Offline vs Online interactions

I think @joenash would actually be a really good "guest" speaker on this, as it seems to work awesomely in [Hackathon Hackers Europe](https://www.facebook.com/groups/hackathonhackerseu/) :smile:

**Comment by ruxiang05** [comment]
I believe _How to involve STEM students into technology_ or something similar would be great.

**Comment by joenash** [comment]
@xoneco I totally missed this, but I can absolutely do that. Next week we have tech writing talks, will look at it for week after! 

@ruxiang05 thanks!

**Comment by DickensA** [comment]
Hey - I posted this in slack finally updating onto here 😸  - it would be great to have a workshop or talk on dispute resolution within your community, it's not something we expect or plan for but sometimes grievances can arise and the way we handle them is really important (speaking from experience) 

A few things to cover here would be: 

Unconscious bias
Code of Conduct - what it actually means and what to do if an incident is reported
Personal disputes amongst a team
Acceptable standards of behaviour in your committee & community

😄 

Thanks

**Comment by PandelisZ** [comment]
I'd like to hear for @PeterDaveHello and how he has time to contribute and maintain so many projects. 

In extension to that, how to get into general OSS development and reading into issues. Also @alyssais is a brew contributor now also would be interesting to hear from the both of them

**Comment by alyssais** [comment]
@PandelisZ wrong repo ;)

**Comment by joenash** [comment]
@PeterDaveHello @alyssais I would welcome both of these ;)

**Comment by alyssais** [comment]
@joenash also wrong repo ;)

**Comment by PandelisZ** [comment]
@alyssais OMG! I knew i posted this. I put this on the real repo and then checked and my browser autocompleted to here. Darn sorry my bad. Also double posted.
